quandl/model/datatable.py

```diff
@@ -14,3 +29,57 @@ def get_path(cls):
     def data(self, **options):
         updated_options = Util.convert_options(**options)
         return Data.page(self, **updated_options)
+
+    def download_file(self, file_or_folder_path, **options):
+        if not isinstance(file_or_folder_path, str):
+            raise QuandlError(Message.ERROR_FOLDER_ISSUE)
+
+        if 'params' not in options:
+            options['params'] = {}
+
+        return self._url_request(file_or_folder_path, **options['params'])
+
+    def _url_request(self, file_or_folder_path, **options):
+        url = self._download_request_path(**options)
+        code_name = self.code
+
+        r = Connection.request('get', url, **options)
+        response_data = r.json()
+
+        status = response_data['datatable_bulk_download']['file']['status']
+
+        if status == 'fresh':
+            file_link = response_data['datatable_bulk_download']['file']['link']
+
+            file_path = file_or_folder_path
+            if os.path.isdir(file_or_folder_path):
+                file_path = os.path.join(file_or_folder_path,
+                                         '{}.{}'.format(code_name.replace('/', '_'), 'zip'))
+
+            res = urlopen(file_link)
+
+            with open(file_path, 'wb') as fd:
+                while True:
+                    chunk = res.read(self.BULK_CHUNK_SIZE)
+                    if not chunk:
+                        break
+                    fd.write(chunk)
+
+            return file_path
+        else:
+            print(Message.LONG_GENERATION_TIME)
+            sleep(30)
```

Comment:
could you put this 30 seconds waiting time into a config file or define it as const variable with meaningful?

Reply:
good idea